### PR TITLE
HTML-753: Messages datetimeAfterVisitDate and datetimeBeforeVisitDate.

### PR DESCRIPTION
**Pull request by JFigueiredo89**
Issue: https://issues.openmrs.org/browse/HTML-753

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/htmlformentryui/fragment/controller/htmlform/EnterHtmlFormFragmentController.java [hunk 1]
```diff
@@ -258,10 +258,10 @@ public SimpleObject submit(UiSessionContext sessionContext,
                 new EncounterDomainWrapper(formEncounter).attachToVisit(visit);
             }
             catch (EncounterDateBeforeVisitStartDateException e) {
-                validationErrors.add(new FormSubmissionError("general-form-error", "Encounter datetime should be after the visit start date"));
+                validationErrors.add(new FormSubmissionError("general-form-error",  Context.getMessageSourceService().getMessage("htmlformentryui.datetimeAfterVisitDate")));
```
Comment (mks-d) [inline omod/src/main/java/org/openmrs/module/htmlformentryui/fragment/controller/htmlform/EnterHtmlFormFragmentController.java:261]: Rather do:
```java
validationErrors.add(new FormSubmissionError("general-form-error",  ui.message("htmlformentryui.datetimeAfterVisitDate")));
```

**Review by mks-d — CHANGES_REQUESTED** [review]

**Review by mks-d — APPROVED** [review]
LGTM, @mogoodrich?

**Review by mogoodrich — APPROVED** [review]
LGTM, thanks!  Just to confirm, were the translations added to Transifex?

**Review by mks-d — APPROVED** [review]
@mogoodrich we will figure out @JFigueiredo89's Transifex access later (as I couldn't even figure out what was happening with mine.)

**Comment by mogoodrich** [comment]
That's fine @mks-d but realize until you get them in Transifex, the next time anyone pulls in translations from Transifex the French and Spanish translations will be overridden with the English text.

Feel free to ping me on Slack with your Transifex username and I can look into making sure you have access.

**Pull request merged**